template/_package.json

```diff
@@ -3,7 +3,12 @@
   "version": "1.0.0",
   "description": "<%= description %>",
   "author": "<%= author %>",
-  "private": true,
+  "private": true, <% if (server === 'feathers') { %>
```

Comment:
The space after `,` can be removed.

Reply:
Unnecessary space removed on commit fd0f9c4eb844e0b26483b6bc9da6f8dd3cc410ee